components/mail/thunderbird/manifests/sample-manifest.p5m

```diff
@@ -22,9026 +22,121 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCrypto.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCycleCollectionNoteChild.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCycleCollectionNoteRootCallback.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCycleCollectionParticipant.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCycleCollectionTraversalCallback.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsCycleCollector.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDBFolderInfo.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMAttributeMap.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMCID.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMCSSAttrDeclaration.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMCSSDeclaration.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMCSSRGBColor.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMClassInfoClasses.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMClassInfoID.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMJSUtils.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMNavigationTiming.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMOfflineResourceList.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDOMString.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDataHashtable.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDateTimeFormatCID.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDebug.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDebugImpl.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeleteDir.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDependentString.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDependentSubstring.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeprecatedOperationList.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeque.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDetectionConfident.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeviceContext.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeviceContextSpecProxy.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeviceSensors.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirPrefs.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirection.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryService.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceAtomList.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceDefs.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceUtils.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDisplayItemTypes.h
-file path=usr/include/thunderbird-$(COMPONEN
```

Comment:
Is the sample-manifest freshly generated? In my version all usr/lib/thunderbird/... paths contain $(MACH64) making them usr/lib/$(MACH64)/thunderbird.
This is also what I expect with BUILD:BITS=64.